src/modules/auth/ForgetPasswordSuccess/ForgetPasswordSuccess.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { GlobalRoute } from '@src/constants/PageRoutes';
+import { VStack, Text, Link } from '@chakra-ui/react';
+import { Button } from '@common/components';
+
+export default function ForgetPasswordSuccess() {
+  const { t } = useTranslation('auth');
+
+  return (
+    <VStack minW="567px" minH="280px" gap="60px" align="center">
+      <VStack align="stretch">
+        <Text fontSize="30px" fontWeight="400">
+          {t('forget-password-success.page.title')}
+        </Text>
+        <Text fontSize="15px" fontWeight="400">
+          {t('forget-password-success.page.subtitle')}
+        </Text>
+      </VStack>
+      <Button type="button">
```

Comment:
Here you can use `onClick=()=>router.push('target-url')`
router comes from `import useRoute from "next/link"`

Reply:
The button should be stretch